vignettes/explanation-node-js.Rmd

```diff
@@ -7,11 +7,37 @@ vignette: >
   %\VignetteEncoding{UTF-8}
 ---
```

Comment:
Explanation: Node.js -->. Explanation: How Rhino builds CSS and JS using Node

Reply:
Expanded the title.